File: two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> vi;
            int a = 0;
            int b;
            vector<int>::iterator it;
            for(int i = 0; i < nums.size(); i++){
                a = target - nums[i];
                it = find(nums.begin(), nums.end(), a);
                b = it-nums.begin();
                    if(it != nums.end()){
                        if(b == i){
                            b = 0;
                            continue;
                        }
                        vi.push_back(i);
                        vi.push_back(it-nums.begin());
                        break;
                    }
                    else continue;           
            }
            
        return vi;
    }
};

File: two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];
        for(int i = 0; i < nums.length; ++i) {
            int first = nums[i];
            int second = target - nums[i];
            int j = i + 1;
            while(j < nums.length) {
                if(nums[j] == second) {
                    result[0] = i;
                    result[1] = j;
                    break;
                }
                j++;
            }
        }
        return result;
    }
}